app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -1,11 +1,11 @@
 = render partial: 'errors/error_summary', locals: { ep: @allocation, error_summary: t('.error_summary') }
 
-.notice-summary.hidden{ role: 'group', 'aria-labelledby': 'notice-summary-heading', tabindex: '-1' }
-  %span.heading-medium.notice-summary-heading{ id: 'notice-summary-heading' }
+.notice-summary.hidden{ role: 'group', 'aria-labelledby': 'notice-summary-heading-id', tabindex: '-1' }
+  %span.heading-medium.notice-summary-heading{ id: 'notice-summary-heading-id' }
```

Comment:
Looks good.
Minor coding style update: `%span#notice-summary-heading-id.heading-medium.notice-summary-heading`

Reply:
that would be inconsistent with the `_flash` notices partial though? which uses id: 
still want to

Reply:
As the ID's aren't dynamic we should add them to haml without the need for braces `{}`

Reply:
will have to change the flash notices too then for consistency - hopefully won't break anything.

have you been able to determine why the axe-matchers are suddenly throwing this error?

Reply:
> will have to change the flash notices too then for consistency - hopefully won't break anything.
> 
> have you been able to determine why the axe-matchers are suddenly throwing this error?

Currently  reviewing it locally

Reply:
actually have left the flash notices alone. you can PR if you'd like - maybe track down some other similar id styling issues, but low priority atm i'd say